### ISSUE TITLE
Investigate disabling memory over commit and/or the OOM killer

### ISSUE THREAD

**Issue by shuhaowu** [ros-realtime/ros-realtime-rpi4-image#38]

**Comment by shuhaowu** [comment]
I think disabling these by default may have some negative side effects for non-RT applications. The benefit for RT application is rather minor, if the RT application is properly coded. Unless there's a good reason to enable this, I'm actually thinking of not enabling this for now. That said, we may consider this again in the future?

**Comment by carlossvg** [comment]
> I think disabling these by default may have some negative side effects for non-RT applications. The benefit for RT application is rather minor, if the RT application is properly coded. Unless there's a good reason to enable this, I'm actually thinking of not enabling this for now. That said, we may consider this again in the future?

@shuhaowu I agree. This was more related to running benchmarks under heavy memory stress but you can always enable it specifically for the tests.
